omod/src/main/java/org/openmrs/module/appframework/rest/ExtensionResource.java

```diff
@@ -104,4 +112,28 @@ protected PageableResult doSearch(RequestContext context) {
         }
         return new NeedsPaging<Extension>(results, context);
     }
+
+    @Override
+    public Model getGETModel(Representation rep) {
```

Comment:
Let's just follow the structure used in `getRepresentationDescription`. It even matches the responses that we got from the test cases. Though it does not comply with FULL >= DEFAULT
```
ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);

modelImpl
                    .property("uuid", new StringProperty())
                    .property("display", new StringProperty());

 if (rep instanceof DefaultRepresentation) {
                    .property("appId", new StringProperty())
                    .property("extensionPointId", new StringProperty())
                    .property("type", new StringProperty())
                    .property("label", new StringProperty())
                    .property("url", new StringProperty())
                    .property("icon", new StringProperty())
                    .property("order", new IntegerProperty())
                    .property("requiredPrivilege", new StringProperty())
                    .property("featureToggle", new StringProperty())
                    .property("require", new StringProperty())
                    .property("script", new StringProperty())
                    .property("extensionParams", new MapProperty(new StringProperty()))
                    .property("belongsTo", new RefProperty());
}

```

@f4ww4z sorry for keep you waiting. I think @dkayiwa will also agree. Make the above changes and then we could merge.